src/pages/NewChairConsole.js

```diff
@@ -1,40 +1,426 @@
-import { Component } from 'react';
-import { a, div } from 'react-hyperscript-helpers';
-import {PageHeading} from "../components/PageHeading";
+import {filter, isEmpty, isNil, find, cloneDeep} from 'lodash/fp';
+import Modal from 'react-modal';
+import { useState, useEffect, useRef} from 'react';
+import { div, h, img } from 'react-hyperscript-helpers';
+// import { SearchBox } from '../components/SearchBox';
+import { DAR, DataSet, PendingCases, User } from '../libs/ajax';
+import { Storage } from '../libs/storage';
+// import { NavigationUtils, USER_ROLES } from '../libs/utils';
+import { DataUseTranslation } from '../libs/dataUseTranslation';
+import {Notifications, formatDate} from '../libs/utils';
+import { Theme } from '../libs/theme';
 
-class NewChairConsole extends Component {
-
-  render() {
-    const titleStyle = {
-      color: '#1F3B50',
+const styles = {
+  PAGE: {
+    width: "90%",
+    margin: "0 auto"
+  },
+  TITLE: {
+    fontFamily: "Montserrat",
+    fontWeight: Theme.font.weight.semibold,
+    fontSize: Theme.font.size.title,
+  },
+  SMALL: {
+    fontFamily: 'Montserrat',
+    fontWeight: Theme.font.weight.regular,
+    fontSize: Theme.font.size.small
+  },
+  HEADER_IMG: {
+    width: '60px',
+    height: '60px',
+  },
+  HEADER_CONTAINER: {
+    display: 'flex',
+    flexDirection: "column"
+  },
+  ICON_CONTAINER: {
+    flexBasis: '76px',
+    height: '60px',
+    paddingRight: '16px'
+  },
+  RIGHT_HEADER_SECTION: {
+    display: 'flex',
+    alignItems: 'flex-end'
+  },
+  LEFT_HEADER_SECTION: {
+    display: 'flex',
+    flexDirection: 'row',
+    paddingTop: "3rem"
+  },
+  TABLE: {
```

Comment:
I'd love for these to be re-usable by other components. Can the table styles be extracted to themes.js?

Reply:
Sure, makes sense to have them as exportable constants there